util/index.ts

```diff
@@ -118,9 +126,10 @@ const handleReputationItems = async (buyer, seller, amount) => {
   }
 };
 
-const getBtcFiatPrice = async (fiatCode, fiatAmount) => {
+const getBtcFiatPrice = async (fiatCode: string, fiatAmount: number) => {
   try {
     const currency = getCurrency(fiatCode);
+    if (!(currency instanceof Object)) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program